index.js

```diff
@@ -34,23 +35,33 @@ function formatBody(body) {
 }
 
 function logRequestResponse(req, proxyResponse, err) {
-    const fileName = `${req.method}-${req.originalUrl}`.replace(/\//g, "|");
-    fs.writeFileSync(`${__dirname}/requests/${fileName}.json`, JSON.stringify({
+    const mappingFileName = `${req.method}|${service}${req.originalUrl}`
+        .replace(/\//g, "_")
+        .replace(/\?/g, "-")
+        .replace("=", "|");
+    const bodyFileName = `${mappingFileName}__Body`;
+
+    fs.writeFileSync(`${__dirname}/requests/${mappingFileName}.json`, JSON.stringify({
         request: {
-            headers: req.headers,
             method: req.method,
-            path: req.originalUrl,
-            body: formatBody(req.rawBody)
+            url: `${service}${req.originalUrl}`
```

Comment:
Should the service be prefixed here?

Reply:
This is to allow the mocking of multiple services with one wiremock?